backend/services/importedStatsService.js

```
@@ -0,0 +1,137 @@
+        // // Store computed daily statistics back to Firestore
+        // const statsCollectionRef = babyRef.collection('importedDailyStatistics');
+        // const batch = db.batch();
+        // for (const [day, stats] of Object.entries(dailyStatistics)) {
+        //     const statsDocRef = statsCollectionRef.doc(day);
+        //     batch.set(statsDocRef, stats);
+        // }
+        // await batch.commit();
+        // console.log('Stored computed daily statistics in Firestore.');
+
+        return { message: 'Successfully computed and stored statistics for imported entries.'};
+        return { message: 'Successfully computed and stored statistics for imported entries.', data: dailyStatistics};
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)